docs/jwst/jump/arguments.rst

```diff
@@ -37,6 +39,7 @@ The ``jump`` step has five optional arguments that can be set by the user:
   the flagging of neighbors of marginal detections. Any primary jump below this value will
   not have its neighbors flagged. The goal is to prevent flagging jumps that would be too
   small to significantly affect the slope determination.  The default value is 10.
+**Parameters that affect after jump Flagging**
```

Comment:
Need a blank line before this new line, to finish off the previous bullet list. This is crashing the docs build.